reactor-netty-http/src/main/java/reactor/netty/http/server/logging/BaseAccessLogHandler.java

```diff
@@ -29,12 +31,19 @@
  */
 class BaseAccessLogHandler extends ChannelDuplexHandler {
 
+	static final DateTimeFormatter DEFAULT_DATE_TIME_FORMAT =
+			DateTimeFormatter.ofPattern("dd/MMM/yyyy:HH:mm:ss Z");
+
```

Comment:
We have already this date_time formatter in `AbstractAccessLogArgProvider`. Do we need it here also?

Reply:
no, not until `String zonedDateTime()` is completely removed. On the second thought - I also don't like that I format access datetime twice per request. Let me revert BaseAccessLogHandler and keep is at it was. Once `String zonedDateTime()` gets removed - formatting will be moved here